lib/pterodactyl_service.php

```diff
@@ -329,6 +329,21 @@ public function getFields($pterodactylEgg, $package, $vars = null, $admin = fals
             }
         }
 
+        // Set js to refetch options when the nest or egg is changed
+        $fields->setHtml("
+            <script type=\"text/javascript\">
+                $(document).ready(function() {
+                    // Re-fetch module options to pull in eggs and egg variables
+                    // when a nest or egg respectively is selected
+                    $('.package_options').on('focusout', '[name^=\"configoptions\"]', function() {
```

Comment:
This reloads whenever any config option is changed.  If possible we'd like to only reload when the location, nest, or egg is changed

Reply:
I've updated this, but could use a second pair of eyes for testing.